### PR TITLE
take advantage of skipNpm for some tests

### PR DESCRIPTION
**Pull request by kellyselden**
cc @simonihmig

### CODE REVIEW

**Comment by kratiahuja** [comment]
@kellyselden what does `skipInstall` do?

**Comment by kellyselden** [comment]
skips npm install if you are going to run `app.run('npm', 'install')` later.

File: test/head-content-test.js [hunk 1]
```diff
@@ -14,7 +14,9 @@ describe('head content acceptance', function() {
   before(function() {
     app = new AddonTestApp();
 
-    return app.create('head-content')
+    return app.create('head-content', {
+      skipInstall: true
```
Comment (simonihmig) [inline test/head-content-test.js]: @kellyselden this should be `skipNpm`! ;)

Comment (kellyselden) [inline test/head-content-test.js]: my stupid brain.

**Comment by kellyselden** [comment]
@kratiahuja Do you have any opinion on this?

**Comment by kratiahuja** [comment]
Nope I think this is fine. Will save test time. Feel free to merge

**Pull request merged**